tests/unit_tests/collection/test_k_ohclv.py

```diff
@@ -0,0 +1,167 @@
+import pytest, requests, re, os
+import numpy as np
+import pandas as pd
+from pandas.util.testing import assert_frame_equal, assert_series_equal
+
+from crypto_analytics.collection.data_source import KrakenOHLCV
+from crypto_analytics.types import Interval
+
+# mock data
+
+k_ohclv_dtypes = {'time': np.int64, 'open': object, 'high': object, 'low': object, 'close': object, 'vwap': object, 'volume': object, 'count': np.int64 }
+k_ohclv_columns = ['time', 'open', 'high', 'low', 'close', 'vwap', 'volume', 'count']
+k_ohclv_success = {'error': [], 'result': {'XXBTZUSD': [[1560123060, '7633.2', '7636.2', '7633.2', '7635.6', '7635.7', '2.23099305', 6]], 'last': 1560123060}}
+k_ohclv_success_df = pd.DataFrame(k_ohclv_success['result']['XXBTZUSD'], columns=k_ohclv_columns).astype(k_ohclv_dtypes)
+k_ohclv_incomplete_candle = {'error': [], 'result': {'XXBTZUSD': [[1560123060, '7633.2', '7636.2', '7633.2', '7635.6', '7635.7', '2.23099305', 6], [1560123120, '7635.6', '7635.6', '7630.5', '7633.1', '7630.6', '0.58258092', 2]], 'last': 1560123060}}
+k_ohclv_incomplete_candle_df = pd.DataFrame(k_ohclv_incomplete_candle['result']['XXBTZUSD'], columns=k_ohclv_columns).astype(k_ohclv_dtypes)
```

Comment:
Importing mock data from `mock_data.py` was getting ridiculous. I think it's better to keep all our test case info on the same file anyway.

Reply:
I'm eventually going to remove the `mock_data.py` file but I think that's out of the scope of this story.